packages/runtime/runtime-definitions/api-report/runtime-definitions.legacy.alpha.api.md

```diff
@@ -105,6 +101,14 @@ export interface IContainerRuntimeBaseEvents extends IEvent {
     (event: "dispose", listener: () => void): any;
 }
 
+// @alpha @sealed @deprecated (undocumented)
+export interface IContainerRuntimeBaseExperimental extends IContainerRuntimeBase {
+    // (undocumented)
+    enterStagingMode?(): StageControlsExperimental;
+    // (undocumented)
+    readonly inStagingMode?: boolean;
```

Comment:
I think we've discussed before, but this will want eventing as observable state that transitions.

Reply:
yeah. it'll come later